code/modules/admin/verbs/adminhelp.dm

```diff
@@ -91,24 +91,47 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+
+	var/datum/adminticket/ticket
+	var/ref_client = "\ref[src]"
+	for(var/I in admintickets)
+		var/datum/adminticket/T = I
+		if(T.permckey == src.ckey && T.resolved == "No")
+			if(alert(usr,"You already have an adminhelp open, would you like to bump it?", "Bump Adminhelp", "Yes", "No") == "Yes")
+				T.logs += "[src.ckey] has bumped this adminhelp!"
+				if(T.admin == "N/A")
+					usr << "<b>Due to the fact your Adminhelp had no assigned admin, admins have been pinged.</b>"
+					message_admins("[src.ckey] has bumped their adminhelp #[T.ID], still no assigned admin!")
+					msg = "<span class='adminnotice'><b><font color='red'>HELP: </font><A HREF='?priv_msg=[ckey];ahelp_reply=1'>[key_name(src)]</A> [ADMIN_QUE(mob)] [ADMIN_PP(mob)] [ADMIN_VV(mob)] [ADMIN_SM(mob)] [ADMIN_FLW(mob)] [ADMIN_TP(mob)] (<A HREF='?_src_=holder;rejectadminhelp=[ref_client]'>REJT</A>) (<A HREF='?_src_=holder;icissue=[ref_client]'>IC</A>) (<A HREF='?_src_=ticket;resolve=[T.ID]'>R</a>):</b> [msg]</span>"
+					for(var/client/X in admins)
+						if(X.prefs.toggles & SOUND_ADMINHELP)
+							X << 'sound/effects/adminhelp.ogg'
+						X << msg
+				else
+					usr << "<b>Admins have been notified.</b>"
+					message_admins("[src.ckey] has bumped their adminhelp #[T.ID].")
+				src.verbs -= /client/verb/adminhelp
+				adminhelptimerid = addtimer(CALLBACK(src, .proc/giveadminhelpverb), 1200, TIMER_STOPPABLE) //2 minute cooldown of admin helps
+				return
+			usr << "<b>Thank you for your patience.</b>"
+			return
+
+	src.verbs -= /client/verb/adminhelp

```

Comment:
This adminhelp verb is not part of a list, and is found naturally already : https://github.com/HippieStation/HippieStation13/blob/master/code/modules/admin/verbs/adminhelp.dm#L102